.github/workflows/e2e-tests.yml

```
@@ -0,0 +1,86 @@
+        pipenv run playwright install-deps
+    
+    - name: Create test environment file
+      run: |
+        cp .env.example .env
+        echo "FALKORDB_HOST=localhost" >> .env
+        echo "FALKORDB_PORT=6379" >> .env
+        echo "FLASK_SECRET_KEY=test-secret-key-for-ci" >> .env
+        echo "FLASK_DEBUG=False" >> .env
+    
+    - name: Wait for FalkorDB
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2046:warning:1:19: Quote this to prevent word splitting

(shellcheck)